README.md

```diff
@@ -40,6 +40,10 @@ Where `type` is one of the following:
 * `style`
 * `test`
 
+Where `flags` is one or more of the following (comma separated):
```

Comment:
Nit: Change it to:

```md
Where `flags` is an optional comma-separated list of one or more of the following (must be surrounded in square brackets):
```

to make it clear that they're optional and that they need to be surrounded by `[]`s.

Reply:
👍